### DISCUSSION TITLE
Raspberry pi password

### DISCUSSION THREAD

**Discussion by Frosty20001** [onlaj/Piano-LED-Visualizer#395] [General]
Hi, im completely oblivious to how to use raspberry pi's at all, im trying to set up my pi using the sd card but its asking for me to insert the default password. Im unable to do this as i cant conenct my keyboard to the it as i dont have a female to female usb cable and cant connect directly as my keyboard isnt micro usb, is there anyway to bypass this?

**Comment by onlaj** [comment]
Hey.
[Configure WI-Fi](https://github.com/onlaj/Piano-LED-Visualizer/blob/master/Docs/wifi_setup.md) and then connect to it through SSH (using [Putty](https://www.putty.org/), for example)

**Reply by Frosty20001** [reply]
hi, i am now able to connect my keyboard to the raspberry pi but i cant seem to get into it still. doesnt seem to have the wifi set up either
![IMG_0311](https://user-images.githubusercontent.com/116835484/198699169-2e7edd90-59c5-4f33-9fdf-c0df18f5ab9b.jpg)
![IMG_0309](https://user-images.githubusercontent.com/116835484/198699175-260b4dd3-7ca4-4681-9114-04509348f5e4.jpg)

I am also unable to enter anything after typing passwd. I can only press enter in which im met with an error message

**Reply by Frosty20001** [reply]
also left my wifi name and password out but they were entered

**Reply by Frosty20001** [reply]
Im also not getting any output to my lcd except as white screen

**Reply by onlaj** [reply]
>doesnt seem to have the wifi set up either

As expected, you need to provide your ssid and password

**Reply by Frosty20001** [reply]
they were entered i just removed them before posting the pic
